Apps/Playground/Win32/App.cpp

```diff
@@ -401,15 +401,60 @@ LRESULT CALLBACK WndProc(HWND hWnd, UINT message, WPARAM wParam, LPARAM lParam)
             SetCapture(hWnd);
```

Comment:
Does SetCapture refcount? Does this work properly if multiple buttons are down?

Reply:
AFAIK, it does not.  I have modified the code to account for this.